cloud_connectors/azure/digital_twins_connector/README.md

```diff
@@ -36,6 +36,24 @@ chmod +x digital_twins_setup.sh
 ./digital_twins_setup.sh -r myRG -l westus2 -d myADT
 ```
 
+### Automated Azure Digital Twins Setup for Smart Trailer Example
+
+Before starting this section, please view [Prerequisites for Automated Deployment of Azure Resources](../README.md#prerequisites-for-automated-deployment-of-azure-resources).
```

Comment:
... please read ...

Reply:
updated in this section and the one I copied from